deps/zlib/contrib/optimizations/insert_string.h

```diff
@@ -16,6 +16,9 @@
 // clang-format off
 #if defined(CRC32_SIMD_SSE42_PCLMUL)
   /* Required to make MSVC bot build pass. */
+  // TODO(raisinten): When https://github.com/nodejs/node/pull/33044 lands,
+  // remove the next line and add `-msse4.2` to the command line options.
```

Comment:
Assuming this lands before #33044, a commit should be added to that PR that resolves this TODO.

Reply:
CC @richardlau
Please consider adding these options in your PR if you haven't added them already. :)